### ISSUE TITLE
[Bug]: Web ePub Reader Incorrect Progress Upon Reading

### ISSUE THREAD

**Issue by ozen-m** [advplyr/audiobookshelf#2376]
### Describe the issue

When continuing to read a book with a remembered progress %, on Edge, Chrome browsers incorrectly continues at a different progress %. For example, reading up to 11%, then exiting, then continuing to read again continues at 6%, instead of 12%.

![image](https://github.com/advplyr/audiobookshelf/assets/74519617/c7e2dd37-7c97-47e6-af40-f2472196a235)
Then exiting, shows progress at 11%
![image](https://github.com/advplyr/audiobookshelf/assets/74519617/5f406548-394c-4c18-83e8-1392c5ab5cf2)
Leads to another progress state
![image](https://github.com/advplyr/audiobookshelf/assets/74519617/2a258e45-4935-4da9-8099-940a6668b27a)
Exiting, which is shown to be at 6% instead of 11%
![image](https://github.com/advplyr/audiobookshelf/assets/74519617/c1afc411-960f-45a5-aa72-abbf903af84a)

Format: ePub
Browsers: Tested on Edge, Chrome - even without extensions

### Steps to reproduce the issue

1. Read up to a certain point (11%)
2. Exit reading, which shows 11% in "Your Progress"
3. Click on "Read" to continue reading at said progress (11%), but book shows different progress
4. Exit reading, immediately without turning pages, which shows 6% instead at "Your Progress"

### Audiobookshelf version

v2.6.0

### How are you running audiobookshelf?

Docker

**Comment by dathbe** [comment]
I'm seeing something similar.  It's not consistent though.  I read a book to a chapter break where the chapter heading appears at the top of the left page.  I then exit the book, and immediately (same browser) click to read the book again.  The reader pops open to the preceding page, where the end of the preceding chapter is on the right page.  I can repeat this a few times and each time the reader backs up another page (well, two pages technically).  Until a certain point where it seems to stick and just stays there.  I'm not entirely sure how an ereader saves progress given that each screen is going to show a different amount of text, but is it possible that ABS is marking progress just a few characters before the page (e.g. white space) and therefore reverting to the previous page on reopen?  And then that doesn't happen sometimes because sometimes there aren't the offending characters at the beginning/end of the page (e.g. no white space)?

**Comment by TheNomad11** [comment]
> I had better consistencies when setting the font size to 100%, but that eliminates changing font size for a more readable experience in exchange for consistent paging when continuing.

Yeah, it is enough to change the font size and I get moved to some pages before I was. The same happens when I go into full screen mode and back. In all cases I end up somewhere else in the book. It is just a few pages difference, so no big deal, though!

**Comment by eranelbaz** [comment]
Seeing the same
On web and android app
